src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,354 @@
+import * as React from 'react';
+import { If, Then, Else } from 'react-if';
+import * as _ from 'lodash';
+import {observer} from "mobx-react";
+import {
+    IClinicalGroupMatch, IGenomicGroupMatch, IGenomicMatch, INctTrial, ITrial,
+    ITrialMatch, IDetailedTrialMatch, IArmMatch
+} from "../../../shared/model/MatchMiner";
+import styles from './style/trialMatch.module.scss';
+import { computed } from "mobx";
+import LazyMobXTable from "../../../shared/components/lazyMobXTable/LazyMobXTable";
+import SampleManager from "../sampleManager";
+import DefaultTooltip, { placeArrowBottomLeft } from "../../../shared/components/defaultTooltip/DefaultTooltip";
+
+export type ITrialMatchProps = {
+    sampleManager?: SampleManager | null;
+    trials: Array<ITrial>;
+    trialMatches:Array<ITrialMatch>;
+    nctTrials:Array<INctTrial>;
+    containerWidth: number;
+}
+
+export type ITrialMatchState = {
+    detailedTrialMatches: Array<IDetailedTrialMatch>;
+}
+
+enum ColumnKey {
+    ID = 'ID',
+    TITLE = 'Title',
+    INTERVENTIONS = 'Interventions',
+    MATCHINGCRITERIA = 'Matching Criteria'
+}
+
+class TrialMatchTableComponent extends LazyMobXTable<IDetailedTrialMatch> {
+
+}
+
+@observer
+export default class TrialMatchTable extends React.Component<ITrialMatchProps, ITrialMatchState> {
+
+    constructor(props: ITrialMatchProps) {
+        super(props);
+        if (props.sampleManager) {
+            this.state = { detailedTrialMatches: this.buildDetailedTrialMatches(props.trialMatches, props.trials, props.nctTrials, props.sampleManager) };
+        } else {
+            this.state = { detailedTrialMatches: this.buildDetailedTrialMatches(props.trialMatches, props.trials, props.nctTrials) };
+        }
+    }
+
+    @computed
+    get columnsWidth() {
+        return {
+            [ColumnKey.ID]: 0.1 * this.props.containerWidth,
+            [ColumnKey.TITLE]: 0.24 * this.props.containerWidth,
+            [ColumnKey.INTERVENTIONS]: 0.16 * this.props.containerWidth,
+            [ColumnKey.MATCHINGCRITERIA]: 0.5 * this.props.containerWidth
+        };
+    }
+
+    private _columns = [{
+        name: ColumnKey.ID,
+        render: (trial: IDetailedTrialMatch) => (
+            <div>
+                <span><a target="_blank" href={"https://www.mskcc.org/cancer-care/clinical-trials/" + trial.protocolNo}>{trial.protocolNo}</a></span><br/>
```

Comment:
I would not use `<br/>`. Let's do a flex container on the top.

Reply:
Removed <br/>